.palantir/revapi.yml

```diff
@@ -1,4 +1,85 @@
 acceptedBreaks:
+  "1.0.0":
+    org.apache.iceberg:iceberg-core:
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.BaseMetadataTable"
+      new: "class org.apache.iceberg.BaseMetadataTable"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.BaseTable"
+      new: "class org.apache.iceberg.BaseTable"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.BaseTransaction.TransactionTable"
+      new: "class org.apache.iceberg.BaseTransaction.TransactionTable"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.MetricsConfig"
+      new: "class org.apache.iceberg.MetricsConfig"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.SerializableTable"
+      new: "class org.apache.iceberg.SerializableTable"
+      justification: "Serialization changed"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.TableMetadata"
+      new: "class org.apache.iceberg.TableMetadata"
+      justification: "Serialization changed"
+    - code: "java.field.noLongerConstant"
+      old: "field org.apache.iceberg.TableProperties.DELETE_MODE_DEFAULT"
+      new: "field org.apache.iceberg.TableProperties.DELETE_MODE_DEFAULT"
+      justification: "Refactorings"
+    - code: "java.field.noLongerConstant"
+      old: "field org.apache.iceberg.TableProperties.MERGE_MODE_DEFAULT"
+      new: "field org.apache.iceberg.TableProperties.MERGE_MODE_DEFAULT"
+      justification: "Refactorings"
+    - code: "java.field.noLongerConstant"
+      old: "field org.apache.iceberg.TableProperties.UPDATE_MODE_DEFAULT"
+      new: "field org.apache.iceberg.TableProperties.UPDATE_MODE_DEFAULT"
+      justification: "Refactorings"
+    - code: "java.method.addedToInterface"
+      new: "method <T extends org.apache.iceberg.rest.RESTResponse> T org.apache.iceberg.rest.RESTClient::delete(java.lang.String,\
+        \ java.util.Map<java.lang.String, java.lang.String>, java.lang.Class<T>, java.util.Map<java.lang.String,\
+        \ java.lang.String>, java.util.function.Consumer<org.apache.iceberg.rest.responses.ErrorResponse>)"
+      justification: "Adding new method where query params can be used"
```

Comment:
I think we should fix this one. This error is correct because existing implementations of `RESTClient` won't have this method. Can we add a version that delegates to the method without query params instead?

Reply:
done

Reply:
I think we should add a check and throw in the event query params are passed to the default implementation.  This shouldn't silently drop query params.